chai_lab/chai1.py

```diff
@@ -294,7 +310,8 @@ def run_inference(
     num_trunk_recycles: int = 3,
     num_diffn_timesteps: int = 200,
     seed: int | None = None,
-    device: str | None = None,
+    device: str | torch.device | None = None,
```

Comment:
Does this play nice with the `typer` interface we've added? Last time I checked, typer is unhappy with `torch.device` type.

Reply:
I don't think it works with typer. Let me revert then